playwright/_impl/_transport.py

```diff
@@ -74,7 +74,7 @@ def serialize_message(self, message: Dict) -> bytes:
             print("\x1b[32mSEND>\x1b[0m", json.dumps(message, indent=2))
         return msg.encode()
 
-    def deserialize_message(self, data: bytes) -> ParsedMessagePayload:
+    def deserialize_message(self, data: Union[str, bytes]) -> ParsedMessagePayload:
```

Comment:
Better to use `AnyStr`

Reply:
I like to be explicit. json.loads accepts `str|byte`

Reply:
Yup Zen of Python